packages/lambda-functions/functions/hello-world/app.ts

```diff
@@ -1,22 +1,120 @@
+import { PrismaClient } from '@translation/db';
 import { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';
+import { bookKeys } from '../../../../data/book-keys';
 
-/**
- *
- * Event doc: https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html#api-gateway-simple-proxy-for-lambda-input-format
- * @param {Object} event - API Gateway Lambda Proxy Input Format
- *
- * Return doc: https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-lambda-proxy-integrations.html
- * @returns {Object} object - API Gateway Lambda Proxy Output Format
- *
- */
+const client = new PrismaClient();
+
+const importServer = 'https://hebrewgreekbible.online';
 
 export const lambdaHandler = async (
```

Comment:
How will we lock this down and make sure that not anyone can trigger an import process?

We can probably setup the AWS stuff so that only our API has permission to access the lambda functions?

Reply:
Yeah, I think that's the easiest approach